framework/src/org/checkerframework/framework/qual/PostconditionAnnotation.java

```diff
@@ -16,6 +16,12 @@
  * <p>The value {@code qualifier} that is necessary for a postcondition specified with {@link
  * EnsuresQualifier} is hard-coded here with the value {@code qualifier}.
  *
+ * <p>Additionally, the elements of the postcondition annotation (annotated by this meta-annotation)
```

Comment:
Please give an example of use to help make the ideas more concrete.
Likewise in PreconditionAnnotation.java.